SurfaceTopography/Uniform/VariableBandwidth.py

```diff
@@ -32,7 +32,30 @@
 from ..HeightContainer import UniformTopographyInterface
 
 
-def checkerboard_detrend(topography, subdivisions):
+def checkerboard_detrend_1D(topography, region_index, order=1, return_plane=False):
```

Comment:
For other functions we used the postfixes `from_area` and `from_profile`. Would this apply here, too?

Reply:
This part of the code has been updated. There is now a `checkerboard_detrend_profile` and `checkerboard_detrend_area` and a `variable_bandwidth_from_area` and `variable_bandwidth_from_profile`. They all should have docstrings.